contents/ServerInstallation.md

```diff
@@ -11,6 +11,19 @@ excerpt: 'Pull and run Exoframe server using docker'
 
 ## Installation and Usage
 
+### Automatic installation
+```
+bash -c "$(curl -fsSL https://raw.github.com/exoframejs/exoframe-server/master/tools/install.sh)"
+```
+Available parameters:
```

Comment:
This should be in a separate section preceded by command to dowload `install.sh` first.